CONTRIBUTING.md

```diff
@@ -11,21 +11,40 @@ We actively welcome your pull requests.
 6. If you haven't already, complete the Contributor License Agreement ("CLA").
 
 ### Forking, cloning and running Rapid
-
 - Make sure to use the latest `node` with something like  `nvm install node`
 - Run `npm run all`
 - Run `npm run quickstart` to start the server
 
 ### Debugging and testing
-
 - In VSCode go to Run>Add Configuration... and select Launch Chrome
 - Examples of breakpoints
 - Conditional breakpoints
 - Some interesting files to break (init, etc)
 
+#### Code walkthrough
```

Comment:
I recommend changing the heading 'Code Walkthrough' to 'Insight into Rapid' for a more fitting and informative title.